jmh-archetypes/jmh-java-benchmark-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -88,6 +88,10 @@ THE POSSIBILITY OF SUCH DAMAGE.
                     <compilerVersion>\${javac.target}</compilerVersion>
                     <source>\${javac.target}</source>
                     <target>\${javac.target}</target>
+                    <compilerArgs>
+                        <arg>-processor</arg>
+                        <arg>org.openjdk.jmh.generators.BenchmarkProcessor</arg>
+                    </compilerArgs>
```

Comment:
Forgot to mention this: this should probably be using `<annotationProcessors>` as well, like the other hunks.